[#16577] Add missing context tags variants & fix API

ulisesmac opened the pull request:
[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

fixes #16577

### Summary

[comment]: # (Summarise the problem and how the pull request solves it)

This PR implements the missing variants of the context-tag variant and matches its API to figma.

![image](https://github.com/status-im/status-mobile/assets/90291778/93b37e74-3644-477a-a806-0721c8dac573)


### Review notes
I noticed some other components are still incomplete:
- `preview-list` lacked of 20 size
- `group-avatar` lacked of 28 size
- `user-avatar` lacked of 28 size

So I implemented them in order to have this PR working and pass design review, but these components should be checked in the same way I did it with this one.

### Testing notes
The previous context-tag uses have been replaced by this new one reworked.

#### Platforms
<!-- (Optional. Specify which platforms should be tested) -->

- Android
- iOS

### Steps to test

- Open Status
- Open quo2.preview -> tags -> context-tags
- Check the variants.


status: ready

ulisesmac committed:
Add size 28 for user-avatar

Add icon-name to group avatar

Add missing context-tags variants and fix the API to match figma

Updates context-tag previous uses

remove `nil nil` params

Rename alias

Remove theme/provider alias

Lint fix

add map style argument to style/container

Fix code style

Update use of group-avatar api

Add issue for multinetwork variant

Add issue for wroing text style

ulisesmac commented on src/quo2/components/avatars/user_avatar/style.cljs:
```diff
@@ -17,6 +17,11 @@
             :status-indicator        12
             :status-indicator-border 2
             :font-size               :paragraph-2}
+   28      {:outer                   28
+            :inner                   28
+            :status-indicator        0
+            :status-indicator-border 0
+            :font-size               :paragraph-2}
```
Add size 28 for user-avatar

ulisesmac commented on src/quo2/components/list_items/preview_list.cljs:
```diff
@@ -21,6 +21,12 @@
        :hole-size     26
        :hole-x        19
        :hole-y        -1}
+   20 {:border-radius {:circular 11 :rounded 8}
+       :hole-radius   {:circular 10 :rounded 9}
+       :margin-left   -4
+       :hole-size     22
+       :hole-x        15
+       :hole-y        -1}
```
Add size 20 for preview-list

ulisesmac commented on src/quo2/components/avatars/group_avatar.cljs:
```diff
@@ -5,17 +5,20 @@
 ;; TODO: this implementation does not support group display picture (can only display default group
 ;; icon).
 (defn group-avatar
   [_]
-  (fn [{:keys [color size]}]
+  (fn [{:keys [color size icon-name]
+        :or   {icon-name :i/group}}]
```
Group-avatar now receives an icon-name to display

ulisesmac commented on src/quo2/components/tags/context_tag/view.cljs:
```diff
@@ -1,105 +1,206 @@
+     [preview-list/preview-list {:type :user :size 20}
+      users]
+
+     :multinetwork ;; TODO: use multinetwork preview-list type when available
+     [preview-list/preview-list {:type :user :size 20}
+      (map #(hash-map :profile-picture %) networks)]
```
the multinetwork component is not available, so we should use it here when it's ready

ulisesmac commented on src/quo2/components/tags/context_tag/view.cljs:
```diff
@@ -1,105 +1,206 @@
+    - community-name
+    - channel-name
+
+  - `:token`
+    - size
+    - token-logo (valid rn/image :source value)
+    - amount
+    - token-name
+
+  - `:network`
+    - size
+    - network-logo (valid rn/image :source value)
+    - network-name
+
+  - `:multinetworks`
+    - networks (vector of {:network-logo pic, :network-name \"a name\"})
+
+  - `:account`
+    - size
+    - account-name
+    - emoji (string containing an emoji)
+
+  - `:collectible`
+    - size
+    - collectible (valid rn/image :source value)
+    - collectible-name
+    - collectible-number
+
+  - `:address`
+    - size
+    - address (string)
+
+  - `:icon`
+    - size
+    - icon
+    - context (string)
+
+  - `:audio`
+    - duration (string)
+  "
```
Long docstring :cry:  but in figma this component has many variants

smohamedjavid replied:
A separate appreciation for the docstring :clap: :clap: :clap:

ulisesmac commented on src/quo2/core.cljs:
```diff
@@ -288,11 +288,8 @@
-(def group-avatar-tag quo2.components.tags.context-tag.view/group-avatar-tag)
-(def audio-tag quo2.components.tags.context-tag.view/audio-tag)
-(def community-tag quo2.components.tags.context-tag.view/community-tag)
+
+(def context-tag quo2.components.tags.context-tag.view/view)
```
Previously we had many context-tag variants exposed, now it is just `context-tag` and we use `:type`

J-Son89 replied:
I had always wondered what these other components were, makes sense now. Thanks @ulisesmac 👍

ulisesmac commented on src/status_im2/contexts/shell/jump_to/components/switcher_cards/view.cljs:
```diff
@@ -34,67 +35,69 @@
-       [:<>]
-
-       nil))])
+  [quo.theme/provider {:theme :dark}
```
Instead of `:override-theme :dark`, I'm using our theme provider and wrapping the components

smohamedjavid replied:
IMO, the component should not be wrapped with the theme provider.

The jump-to screen has the theme provider `:dark`. I believe we use it as this switcher card is used only in "Jump to" screen.
https://github.com/status-im/status-mobile/blob/625510d6c96e0216e787aa225d07209673c8f355/src/status_im2/contexts/shell/jump_to/components/jump_to_screen/view.cljs#L102

ulisesmac replied:
Fixed!

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 661ddcea | [#2](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-16962/2/) | 2023-08-09 23:43:14 | ~5 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-230809-233734-661ddc-pr16962-x86.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-230809-233734-661ddc-pr16962-x86.apk)|

J-Son89 commented on src/quo2/components/tags/context_tag/style.cljs:
```diff
@@ -37,21 +17,88 @@
+                           (not (#{:account :collectible} type)) 16
+                           (= size 24)                           8
+                           :else                                 10)
+        border-color     (colors/custom-color-by-theme customization-color 50 60 nil nil theme)]
```
imo this function is too messy with `nil`, `nil` params. 
I think you should use 
```
(colors/theme-colors 
(colors/custom-color customization-color 50)
(colors/custom-color customization-color 60)
theme)
```
and eventually we should just make the `50` and `60` based on the theme like we discussed in Porto - and use the overlay for customisation pressables. 
This will make this function much simpler. Also the opacity set in 99% of cases is the same so it's unnecessary for dark and light opacity imo.

Anyway, beyond this pr but we should address this at some point soon 👍

ulisesmac replied:
Nice suggestion.

And yes! we should address it soon, since the code is growing more and more

J-Son89 commented on src/status_im2/contexts/syncing/syncing_instructions/view.cljs:
```diff
@@ -41,11 +42,13 @@
-     (resources/get-mock-image (:source value))
-     (i18n/label (:label value))]))
+
+    [quo.theme/provider {:theme :dark}
```
hmm, why are you using the theme provider here? Is there another way to do this? @smohamedjavid ? 🤔 I think you can pass theme to bottom sheet

smohamedjavid replied:
I believe we don't need a theme provider there. The bottom sheet/screen is already wrapped with the theme provider, and the `:dark` theme is already added to this bottom sheet/screen. 

https://github.com/status-im/status-mobile/blob/625510d6c96e0216e787aa225d07209673c8f355/src/status_im2/navigation/screens.cljs#L96-L102

https://github.com/status-im/status-mobile/blob/625510d6c96e0216e787aa225d07209673c8f355/src/status_im2/navigation/options.cljs#L103-L108

ulisesmac replied:
Fixed!

smohamedjavid commented on src/quo2/components/tags/context_tag/view.cljs:
```diff
@@ -1,105 +1,206 @@
             [quo2.components.markdown.text :as text]
             [quo2.components.tags.context-tag.style :as style]
+            [quo2.foundations.colors :as colors]
+            [quo2.theme :as theme]
```
Please alias to `quo.theme` to prevent mixing up with the actual `theme` key.

ulisesmac replied:
Done!

smohamedjavid reviewed (commented):
Nice work! 🚀 @ulisesmac

ulisesmac commented:
@smohamedjavid @J-Son89 I have pushed changes according to your comments.

J-Son89 commented on src/quo2/components/avatars/group_avatar/view.cljs:
```diff
@@ -7,24 +7,27 @@
 
 (def sizes
   {:icon      {:small  12
+               28      16
```
how possible would it be to update the other uses here and fix across the app so we're not mixing semantic names with integers? 🤔

ulisesmac replied:
As I wrote in this PR description:

> So I implemented them in order to have this PR working and pass design review, but these components should be checked in the same way I did it with this one.

I meant, we need to refactor, add the missing sizes and fix the API of these components.

J-Son89 replied:
Thanks, my bad for missing that!

J-Son89 commented on src/quo2/components/avatars/group_avatar/view.cljs:
```diff
@@ -7,24 +7,27 @@
 (defn- view-internal
   [_]
-  (fn [{:keys [color size theme]}]
+  (fn [{:keys [color size theme icon-name]
```
can we change `color` to `customization-color`? or is that a follow up?

ulisesmac replied:
This PR already has a lot of changes related to the context-tag and its uses, adding even more to refactor surrounding components is too mucho IMO. I just touched the other components to add the required features, a separate issue for each one is required. I will create the issues

J-Son89 replied:
Yep I agree, thanks!

J-Son89 commented on src/quo2/components/tags/context_tag/view.cljs:
```diff
@@ -1,105 +1,206 @@
-     :i/play
-     {:color style/audio-tag-icon-color
-      :size  12}]]
+  (str (subs pk 0 5) "..." (subs pk (- (count pk) 3))))
```
hmm, it's okay to change the length of `trim-public-key` method? 🤔

ulisesmac replied:
I think designs have changed, so now this is matching designs

J-Son89 commented on src/quo2/components/tags/context_tag/view.cljs:
```diff
@@ -1,105 +1,206 @@
-   avatar
-   community-name])
+    {:style  (style/text theme)
+     :weight :monospace ;; TODO: fix this style
```
please create an issue and link the TODO here, if it's an old TODO then just remove it 👍

ulisesmac replied:
I will create an issue

J-Son89 commented on src/quo2/components/tags/context_tag/view.cljs:
```diff
@@ -1,105 +1,206 @@
+     [preview-list/preview-list {:type :user :size 20}
+      users]
+
+     :multinetwork ;; TODO: use multinetwork preview-list type when available
```
same for this TODO - is there a corresponding issue? 🤔

ulisesmac replied:
I'm not sure :thinking:  I will check and create it if it doesn't exist

J-Son89 reviewed (approved):
pr looks good, left a few comments if you can take a look at them 👍

vkjr commented on src/quo2/components/avatars/group_avatar/view.cljs:
```diff
@@ -7,24 +7,27 @@
 ;; TODO: this implementation does not support group display picture (can only display default group
 ;; icon).
 (defn- view-internal
   [_]
```
I don't think there is any reason to use form-2 (function that returns function) reagent component here. 
We don't introduce any component state, so form-1 (function that returns data) is enough.

ulisesmac replied:
@vkjr I agree, but I'm not touching this component more than required, if I'd refactor this component it would imply even more changes and this PR is long enough.

as I said in this PR description:
> So I implemented them in order to have this PR working and pass design review, but these components should be checked in the same way I did it with this one.

So I think a specific issue should be created to fix all these problems.

vkjr replied:
@ulisesmac, I see, no worries) I just comment everything that is close enough to the changes you've made :))

vkjr commented on src/quo2/components/list_items/preview_list.cljs:
```diff
@@ -47,11 +54,10 @@
                                      :border-radius border-radius}}]))
 
 (defn list-item
```
Let's use `defn-` to articulate that this is an internal component

ulisesmac replied:
@vkjr same as before, I'm not in the task of refactoring all the surrounding components, that'd imply many more changes.

vkjr commented on src/quo2/components/list_items/preview_list.cljs:
```diff
@@ -47,11 +54,10 @@
                                      :border-radius border-radius}}]))
 
 (defn list-item
-  [index type size item list-size margin-left
```
I'd suggest using a single map argument with keys `:index`, `:type`, `:size`... etc. here. Because with this amount of arguments it becomes harder to maintain they order)

ulisesmac replied:
@vkjr Again, I agree with your suggestion, but I didn't change this function signature. That's not the scope of this PR :+1:

vkjr commented on src/quo2/components/tags/context_tag/style.cljs:
```diff
@@ -37,21 +17,91 @@
-  {:padding-vertical 2
-   :padding-left     2})
+(defn container
+  [theme type size state blur? customization-color]
```
Single map argument can also be used here

ulisesmac replied:
I know, but I prefered to have  a more concise call, also this style function is only used once :thinking: 
 I changed it but still don't like too much the change:

![image](https://github.com/status-im/status-mobile/assets/90291778/0e42024b-7356-457b-b679-1b872aadb07c)

vkjr commented:
It seems that `blur?` option is not taken into account for text color

![IMG_7578 - 01](https://github.com/status-im/status-mobile/assets/5786310/fb212734-de81-41d9-a29d-8e25e74b7c2b)

vkjr commented:
When I look at the component in figma, it is obvious that two possible sizes are 24 and 32. But when I'm going to use the component as a developer, there is no way to know that the second acceptable option is 32. 
If I understood correctly, `style/container` will accept any `size` and will use it for `:height`, but all other styles will check `(if (= size 24) ..)` and fallback for 32 in `else` case which will result in an inconsistent visual look.

I suggest making two possible sizes more articulated in the code. For example, we can accept `:s` and `:m` as values for `:size` prop, as options for 24 and 32 respectively. And in all places where style depends on size, we can use `(cond)` instead of `(if)` to clearly show the difference. 
I believe that also will make the code less fragile if figma designs will change.

@ulisesmac, @J-Son89, wdyt?

vkjr reviewed (changes requested):
Thanks for the PR! Left few comments)

ulisesmac commented:
> When I look at the component in figma, it is obvious that two possible sizes are 24 and 32. But when I'm going to use the component as a developer, there is no way to know that the second acceptable option is 32. 
> If I understood correctly, `style/container` will accept any `size` and will use it for `:height`, but all other styles will check `(if (= size 24) ..)` and fallback for 32 in `else` case which will result in an inconsistent visual look.
> 
> I suggest making two possible sizes more articulated in the code. For example, we can accept `:s` and `:m` as values for `:size` prop, as options for 24 and 32 respectively. And in all places where style depends on size, we can use `(cond)` instead of `(if)` to clearly show the difference. 
> I believe that also will make the code less fragile if figma designs will change.
> 
> @ulisesmac, @J-Son89, wdyt?


@vkjr 
I thought the same and already mentioned in another PR to @J-Son89 and Icaro (I think),
because it would be hard to know which size is expected from the component and it could receive any number, but we agreed to not rename sizes, and when a screen is implemented, the developer should look at the size specified in figma to use a component.

I was also thinking about using `assert` on the params to prevent a wrong usage and maybe turn off the assert check in prod. But tbh, I think it is the same.

ulisesmac commented:
> It seems that `blur?` option is not taken into account for text color
> 
> ![IMG_7578 - 01](https://user-images.githubusercontent.com/5786310/260484052-fb212734-de81-41d9-a29d-8e25e74b7c2b.jpg)

`blur?` is not relevant for text color, the text color depends on the theme, as you can see in these variants, the text color is black when theme is light despite it's `blur?` or not. Same for text being white  when theme is dark.

![image](https://github.com/status-im/status-mobile/assets/90291778/74c20aa4-e829-4f17-bffb-86e9af152f5f)

vkjr commented:
> `blur?` is not relevant for text color, the text color depends on the theme, as you can see in these variants, the text color is black when theme is light despite it's `blur?` or not. Same for text being white when theme is dark.

@ulisesmac, yep, you are right) It is the dark-blur background in quo preview that made me think that something is wrong. But implementation is correct!

ulisesmac commented:
> > `blur?` is not relevant for text color, the text color depends on the theme, as you can see in these variants, the text color is black when theme is light despite it's `blur?` or not. Same for text being white when theme is dark.
> 
> @ulisesmac, yep, you are right) It is the dark-blur background in quo preview that made me think that something is wrong. But implementation is correct!



> > `blur?` is not relevant for text color, the text color depends on the theme, as you can see in these variants, the text color is black when theme is light despite it's `blur?` or not. Same for text being white when theme is dark.
> 
> @ulisesmac, yep, you are right) It is the dark-blur background in quo preview that made me think that something is wrong. But implementation is correct!

Oh yes!, on Android the light blur looks better than on iOS :thinking: ,. I have just tested on iOS and it looks weird :thinking:

vkjr commented:
> @vkjr I thought the same and already mentioned in another PR to @J-Son89 and Icaro (I think), because it would be hard to know which size is expected from the component and it could receive any number, but we agreed to not rename sizes, and when a screen is implemented, the developer should look at the size specified in figma to use a component.
> 
> I was also thinking about using `assert` on the params to prevent a wrong usage and maybe turn off the assert check in prod. But tbh, I think it is the same.

Nice to know it is already discussed) Still would prefer `cond` over `if`s to show all options explicitly. But it is on your decision)

ulisesmac commented:
> When I open the preview screen the first time, there is no icon inside component. But if I change the size to 24 and then to 32 - it appears. Could you please check)
> 
> ![IMG_1685181FACE8-1](https://user-images.githubusercontent.com/5786310/260485169-f04cc5bf-452c-4f72-b7e0-f428b878482f.jpg)

@codemaster115 Has fixed the component and I reabased this branch, so now that's working correctly :smile:

ulisesmac commented:
@J-Son89 @vkjr 
thanks for your reviews. I have addressed your comments. Also I created some issues and menitoned in the `TODO` comments.

- #17008
- #17009 
- #17010

ulisesmac commented:
@Francesca-G
 Can I get a design review for this component?
Thanks!

ulisesmac commented:
After rebasing with develop, the variants relying on the `user-avatar` component:
- Default
- Multiuser
- Multinetwork

are broken, I noticed also the `preview-list` component is broken (which is used for multiuser and multinetwork).
Instead of try to fix the `user-avatar` component in this PR, I opened an issue to address it and once it is solved in the entire app, use it for context-tags.

issue:
- https://github.com/status-im/status-mobile/issues/17055

Francesca-G reviewed (commented):
Looks like the light blur theme is missing

ulisesmac commented:
> Looks like the light blur theme is missing

@Francesca-G The preview screen blur was not looking nice on iOS but on Android it looks better.
I fixed the preview blur bg for light theme on IOS, so could you please test it again, please?

Francesca-G reviewed (approved):
@ulisesmac all good :)

status-im-auto commented:
## 81% of end-end tests have passed
```
Total executed tests: 43
Failed tests: 8
Passed tests: 35
```
```
IDs of failed tests: 702732,702948,702783,703202,702786,702731,702808,702958 
```
<h3>Failed tests (8)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=develop&PR_ID=16962&TR_CASE_IDS=702732,702948,702783,703202,702786,702731,702808,702958">Rerun failed tests</a></li><br/><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220564">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><p><blockquote><div>Device 2: Click until `ActivityCenterElement` by `xpath`: `//*[contains(@text, 'user1')]/ancestor::*[@content-desc='activity']` will be presented </div><div>Device 2: Find `Button` by `accessibility id`: `open-activity-center-button` </div></blockquote></p>

```
medium/test_activity_center.py:315: in test_activity_center_admin_notification_accept_swipe
    self.home_2.clear_chat_long_press(self.username_1)
../views/home_view.py:519: in clear_chat_long_press
    self.get_chat(username).long_press_element()
../views/home_view.py:346: in get_chat
    chat_in_ac.wait_for_element(20)
../views/base_element.py:122: in wait_for_element
    raise TimeoutException(
 Device `2`: `ActivityCenterElement` by` xpath`: `//*[contains(@text, 'user1')]/ancestor::*[@content-desc='activity']` is not found on the screen after wait_for_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2504d974b3004c749692f47dec9405e1?auth=8ee7ae7821a086fa850db735cb61da2c#307">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/46ee6ce7a54e49308313a4aa19931c9f?auth=d4bbf554858291e0353262565bcf46d5#146">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220539">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//*[starts-with(@text,'test message')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']//*[@content-desc='message-status']/android.widget.TextView` </div><div>Device 2: `Text` is `Delivered` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:1377: in test_1_1_chat_is_shown_message_sent_delivered_from_offline
    self.errors.verify_no_errors()
base_test_case.py:177: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Message was not delivered after resending from offline
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d4c6e948d05e4ba48f082a837aaab203?auth=e9d53afe5122898234a4e9d04f71e5bf#457">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bd1b6f48976341e1b8259d5d0eb69d41?auth=2571e1a0a1d35ff725090983583f55a2#403">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220533">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td>

```
Test is not run, e2e blocker  
```

<b>[[reason: [NOTRUN] Pin feature is in development]]</b><br/><br/></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220534">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td>

```
Test is not run, e2e blocker  
```

<b>[[reason: [NOTRUN] Pin feature is in development]]</b><br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220568">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `authors-for-reaction-5` </div><div>Device 1: Tap on found: Button </div></blockquote></p>

```
critical/chats/test_group_chat.py:344: in test_group_chat_reactions
    self.errors.verify_no_errors()
base_test_case.py:177: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Incorrect reactions count for user admin after changing the reactions
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/65bcaf49e2b74e279755034d23da4c22?auth=750cf432458f54a2ede7d08af44bebb0#145">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e8b6a6531ed24f1d95361147835130f2?auth=55df447da12528b72c3eecae78fbc063#98">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/73ad4e1d06bb4f7aa0be3b89ece5abc7?auth=cd56581658a2b360b34c7247c7c0318c#78">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220543">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><p><blockquote><div>Device 3: Looking for a message by text: message from old member </div><div>Device 3: Looking for a message by text: message from new member </div></blockquote></p>

```
critical/chats/test_group_chat.py:441: in test_group_chat_offline_pn
    self.errors.verify_no_errors()
base_test_case.py:177: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Messages PN was not fetched from offline
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/65bcaf49e2b74e279755034d23da4c22?auth=750cf432458f54a2ede7d08af44bebb0#293">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e8b6a6531ed24f1d95361147835130f2?auth=55df447da12528b72c3eecae78fbc063#197">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/73ad4e1d06bb4f7aa0be3b89ece5abc7?auth=cd56581658a2b360b34c7247c7c0318c#174">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220562">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><p><blockquote><div># STEP: Device 1 sends a message with hashtag in the dogs channel</div><div>Device 1: Sending message '#cats' </div></blockquote></p>

```
critical/test_public_chat_browsing.py:1195: in test_community_hashtag_links_to_community_channels
    self.channel_1.send_message(message_with_hashtag)
../views/chat_view.py:988: in send_message
    self.chat_message_input.wait_for_element(wait_chat_input_sec)
../views/base_element.py:122: in wait_for_element
    raise TimeoutException(
 Device `1`: `ChatMessageInput` by` accessibility id`: `chat-message-input` is not found on the screen after wait_for_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/68dbbb20b7a249adabf5ef40c4a3e4b3?auth=652bf0eccc1df9b06fc5d59e3964f4f1#714">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a4a533615efb433ab0c1888cd48e623d?auth=6885e639fdef1fa656d605b3ab92ae19#629">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220541">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><p><blockquote><div># STEP: Invited member gets push notification with the mention and tap it</div><div>Device 2: Getting PN by 'user_2' </div></blockquote></p>

```
critical/test_public_chat_browsing.py:1088: in test_community_mentions_push_notification
    self.errors.verify_no_errors()
base_test_case.py:177: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Push notification with the mention was not received by admin
E    Can not edit a message with a mention
E    Push notification with the mention was not received by the invited member 
```

<b>[[Issue with username in PN, issue #6 in https://github.com/status-im/status-mobile/issues/15500]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/68dbbb20b7a249adabf5ef40c4a3e4b3?auth=652bf0eccc1df9b06fc5d59e3964f4f1#531">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a4a533615efb433ab0c1888cd48e623d?auth=6885e639fdef1fa656d605b3ab92ae19#438">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (35)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220572">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/65bcaf49e2b74e279755034d23da4c22?auth=750cf432458f54a2ede7d08af44bebb0#344">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e8b6a6531ed24f1d95361147835130f2?auth=55df447da12528b72c3eecae78fbc063#222">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/73ad4e1d06bb4f7aa0be3b89ece5abc7?auth=cd56581658a2b360b34c7247c7c0318c#205">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220569">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/65bcaf49e2b74e279755034d23da4c22?auth=750cf432458f54a2ede7d08af44bebb0#250">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e8b6a6531ed24f1d95361147835130f2?auth=55df447da12528b72c3eecae78fbc063#162">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/73ad4e1d06bb4f7aa0be3b89ece5abc7?auth=cd56581658a2b360b34c7247c7c0318c#127">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220542">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/65bcaf49e2b74e279755034d23da4c22?auth=750cf432458f54a2ede7d08af44bebb0#125">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e8b6a6531ed24f1d95361147835130f2?auth=55df447da12528b72c3eecae78fbc063#71">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/73ad4e1d06bb4f7aa0be3b89ece5abc7?auth=cd56581658a2b360b34c7247c7c0318c#71">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220544">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/68dbbb20b7a249adabf5ef40c4a3e4b3?auth=652bf0eccc1df9b06fc5d59e3964f4f1#586">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a4a533615efb433ab0c1888cd48e623d?auth=6885e639fdef1fa656d605b3ab92ae19#498">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220567">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/68dbbb20b7a249adabf5ef40c4a3e4b3?auth=652bf0eccc1df9b06fc5d59e3964f4f1#199">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a4a533615efb433ab0c1888cd48e623d?auth=6885e639fdef1fa656d605b3ab92ae19#152">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220557">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/68dbbb20b7a249adabf5ef40c4a3e4b3?auth=652bf0eccc1df9b06fc5d59e3964f4f1#224">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a4a533615efb433ab0c1888cd48e623d?auth=6885e639fdef1fa656d605b3ab92ae19#209">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220548">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/68dbbb20b7a249adabf5ef40c4a3e4b3?auth=652bf0eccc1df9b06fc5d59e3964f4f1#271">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a4a533615efb433ab0c1888cd48e623d?auth=6885e639fdef1fa656d605b3ab92ae19#276">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220565">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/68dbbb20b7a249adabf5ef40c4a3e4b3?auth=652bf0eccc1df9b06fc5d59e3964f4f1#496">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a4a533615efb433ab0c1888cd48e623d?auth=6885e639fdef1fa656d605b3ab92ae19#411">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220559">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/68dbbb20b7a249adabf5ef40c4a3e4b3?auth=652bf0eccc1df9b06fc5d59e3964f4f1#359">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a4a533615efb433ab0c1888cd48e623d?auth=6885e639fdef1fa656d605b3ab92ae19#344">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220547">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/68dbbb20b7a249adabf5ef40c4a3e4b3?auth=652bf0eccc1df9b06fc5d59e3964f4f1#178">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a4a533615efb433ab0c1888cd48e623d?auth=6885e639fdef1fa656d605b3ab92ae19#123">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220546">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/68dbbb20b7a249adabf5ef40c4a3e4b3?auth=652bf0eccc1df9b06fc5d59e3964f4f1#136">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a4a533615efb433ab0c1888cd48e623d?auth=6885e639fdef1fa656d605b3ab92ae19#94">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220551">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/68dbbb20b7a249adabf5ef40c4a3e4b3?auth=652bf0eccc1df9b06fc5d59e3964f4f1#292">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a4a533615efb433ab0c1888cd48e623d?auth=6885e639fdef1fa656d605b3ab92ae19#291">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220550">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/68dbbb20b7a249adabf5ef40c4a3e4b3?auth=652bf0eccc1df9b06fc5d59e3964f4f1#154">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a4a533615efb433ab0c1888cd48e623d?auth=6885e639fdef1fa656d605b3ab92ae19#111">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220552">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/68dbbb20b7a249adabf5ef40c4a3e4b3?auth=652bf0eccc1df9b06fc5d59e3964f4f1#711">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a4a533615efb433ab0c1888cd48e623d?auth=6885e639fdef1fa656d605b3ab92ae19#598">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>12. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220549">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/68dbbb20b7a249adabf5ef40c4a3e4b3?auth=652bf0eccc1df9b06fc5d59e3964f4f1#333">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a4a533615efb433ab0c1888cd48e623d?auth=6885e639fdef1fa656d605b3ab92ae19#334">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220566">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/64e452162afb4570abdfcf14a5c56c50?auth=2de6e39e7093056cd11e9ad53928b5b4#254">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220536">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/64e452162afb4570abdfcf14a5c56c50?auth=2de6e39e7093056cd11e9ad53928b5b4#115">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220574">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/64e452162afb4570abdfcf14a5c56c50?auth=2de6e39e7093056cd11e9ad53928b5b4#54">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220558">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/64e452162afb4570abdfcf14a5c56c50?auth=2de6e39e7093056cd11e9ad53928b5b4#140">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220553">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/64e452162afb4570abdfcf14a5c56c50?auth=2de6e39e7093056cd11e9ad53928b5b4#80">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220570">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/64e452162afb4570abdfcf14a5c56c50?auth=2de6e39e7093056cd11e9ad53928b5b4#159">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220563">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2504d974b3004c749692f47dec9405e1?auth=8ee7ae7821a086fa850db735cb61da2c#270">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/46ee6ce7a54e49308313a4aa19931c9f?auth=d4bbf554858291e0353262565bcf46d5#128">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220560">test_navigation_jump_to</a>, id: 702936 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2504d974b3004c749692f47dec9405e1?auth=8ee7ae7821a086fa850db735cb61da2c#134">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/46ee6ce7a54e49308313a4aa19931c9f?auth=d4bbf554858291e0353262565bcf46d5#109">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220561">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2504d974b3004c749692f47dec9405e1?auth=8ee7ae7821a086fa850db735cb61da2c#155">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/46ee6ce7a54e49308313a4aa19931c9f?auth=d4bbf554858291e0353262565bcf46d5#112">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220538">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d4c6e948d05e4ba48f082a837aaab203?auth=e9d53afe5122898234a4e9d04f71e5bf#128">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bd1b6f48976341e1b8259d5d0eb69d41?auth=2571e1a0a1d35ff725090983583f55a2#108">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220535">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d4c6e948d05e4ba48f082a837aaab203?auth=e9d53afe5122898234a4e9d04f71e5bf#397">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bd1b6f48976341e1b8259d5d0eb69d41?auth=2571e1a0a1d35ff725090983583f55a2#333">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220540">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d4c6e948d05e4ba48f082a837aaab203?auth=e9d53afe5122898234a4e9d04f71e5bf#579">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bd1b6f48976341e1b8259d5d0eb69d41?auth=2571e1a0a1d35ff725090983583f55a2#552">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220545">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d4c6e948d05e4ba48f082a837aaab203?auth=e9d53afe5122898234a4e9d04f71e5bf#259">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bd1b6f48976341e1b8259d5d0eb69d41?auth=2571e1a0a1d35ff725090983583f55a2#197">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220537">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d4c6e948d05e4ba48f082a837aaab203?auth=e9d53afe5122898234a4e9d04f71e5bf#190">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bd1b6f48976341e1b8259d5d0eb69d41?auth=2571e1a0a1d35ff725090983583f55a2#149">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220573">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d4c6e948d05e4ba48f082a837aaab203?auth=e9d53afe5122898234a4e9d04f71e5bf#511">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bd1b6f48976341e1b8259d5d0eb69d41?auth=2571e1a0a1d35ff725090983583f55a2#539">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220556">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d4c6e948d05e4ba48f082a837aaab203?auth=e9d53afe5122898234a4e9d04f71e5bf#310">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bd1b6f48976341e1b8259d5d0eb69d41?auth=2571e1a0a1d35ff725090983583f55a2#230">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220571">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d4c6e948d05e4ba48f082a837aaab203?auth=e9d53afe5122898234a4e9d04f71e5bf#337">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bd1b6f48976341e1b8259d5d0eb69d41?auth=2571e1a0a1d35ff725090983583f55a2#272">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220532">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d4c6e948d05e4ba48f082a837aaab203?auth=e9d53afe5122898234a4e9d04f71e5bf#78">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bd1b6f48976341e1b8259d5d0eb69d41?auth=2571e1a0a1d35ff725090983583f55a2#77">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220555">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce4f8433c807425a98f4b36bfb4c9d79?auth=e59fbe1ea9b52fe494587f29c7ae0e0f#90">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4f2da935ac9345e89170bf6c8f514e03?auth=134dcd8467b0a2164e1512ba834e30bf#91">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3220554">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce4f8433c807425a98f4b36bfb4c9d79?auth=e59fbe1ea9b52fe494587f29c7ae0e0f#55">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4f2da935ac9345e89170bf6c8f514e03?auth=134dcd8467b0a2164e1512ba834e30bf#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

J-Son89 commented:
@VolodLytvynenko, @ulisesmac is on pto so I have taken over this branch. looks like you are in testing with it so let me know if any issues and I will address them :)

J-Son89 commented:
@VolodLytvynenko, @ulisesmac is on pto so I have taken over this branch. looks like you are in testing with it so let me know if any issues and I will address them :)

VolodLytvynenko commented:
Hi @ulisesmac thank you for PR. No issues from my side. PR can be merged
